### PR TITLE
fix(frontend): Parameter field is missing after choosing another v2 pipeline

### PR DESCRIPTION
**Pull request by jlyaoyuli**
After the concept of `default_version` is removed in v2 API, listPipelineVersions() need to be called after pipeline is selected to retrieve the latest pipeline version.

Pipeline with parameter:

https://user-images.githubusercontent.com/56132941/233697396-0d1aff73-1954-4d2c-8fdc-3ecef900f8e1.mov

Parameter field is missing:

https://user-images.githubusercontent.com/56132941/233697731-286df3c8-43ab-4bad-9301-a95001795b56.mov

After this PR:

https://user-images.githubusercontent.com/56132941/233698044-ca7be78d-a77a-4465-aec1-77cebf09a71a.mov

### CODE REVIEW

File: frontend/src/pages/NewRunV2.tsx [hunk 1]
```diff
@@ -134,6 +134,22 @@ function getPipelineDetailsUrl(
...
+      pipelineId,
+      undefined,
+      1, // Only need the latest one
+      'created_at desc',
+    );
+    return listVersionsResponse.pipeline_versions
+      ? listVersionsResponse.pipeline_versions[0]
+      : undefined;
+  } catch (err) {
+    return;
```
Comment (chensun) [inline frontend/src/pages/NewRunV2.tsx:149]: Should we properly handle the error?

Comment (jlyaoyuli) [reply frontend/src/pages/NewRunV2.tsx:149]: I think we can either return or pop some error message. Let me add a logger for handling this error.

**Review by chensun — APPROVED** [review]
/lgtm
/approve

**Pull request merged**